[AOT Refactor] logging utils

jon-chuang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #114643
* #114564
* #114563
* #114562
* #114561
* #114559
* #114558
* #114557
* #114556
* #114555
* #114554
* #114553
* #114552
* __->__ #114551
* #114550

---

Part _ of https://github.com/pytorch/pytorch/issues/114548

jon-chuang committed:
[AOT Refactor] logging utils

[ghstack-poisoned]

jon-chuang committed:
Update on "[AOT Refactor] logging utils"




[ghstack-poisoned]

jon-chuang committed:
Update on "[AOT Refactor] logging utils"


---

Part _ of https://github.com/pytorch/pytorch/issues/114548



[ghstack-poisoned]

jon-chuang committed:
Update on "[AOT Refactor] logging utils"


---

Part _ of https://github.com/pytorch/pytorch/issues/114548



[ghstack-poisoned]

jon-chuang committed:
Update on "[AOT Refactor] logging utils"


---

Part _ of https://github.com/pytorch/pytorch/issues/114548



[ghstack-poisoned]

jon-chuang committed:
Update on "[AOT Refactor] logging utils"


---

Part _ of https://github.com/pytorch/pytorch/issues/114548



[ghstack-poisoned]

jon-chuang commented:
I think merge failure is unrelated as all other PRs in the stack do not encounter this error

jon-chuang committed:
Update on "[AOT Refactor] logging utils"


---

Part _ of https://github.com/pytorch/pytorch/issues/114548



[ghstack-poisoned]

jon-chuang committed:
Update on "[AOT Refactor] logging utils"


---

Part _ of https://github.com/pytorch/pytorch/issues/114548



[ghstack-poisoned]

bdhirsh commented on torch/_functorch/_aot_autograd/logging_utils.py:
```diff
@@ -0,0 +1,131 @@
+model_name: str = "model"
+
+
+def set_model_name(name):
```
I see this isn't used anywhere - code-motion-only PR is great but maybe add a comment that we should just delete this later

jon-chuang replied:
It's actually a public-facing API

jon-chuang replied:
It's used elsewhere in torch

bdhirsh replied:
yep you're right :( (used in dynamo)

thiagocrepaldi reviewed (commented):
@kit1980 do you know why the `release notes: onnx` label was applied here? I've been noticing this behavior in several PRs lately

jon-chuang committed:
Update on "[AOT Refactor] logging utils"


---

Part _ of https://github.com/pytorch/pytorch/issues/114548



[ghstack-poisoned]

jon-chuang committed:
Update on "[AOT Refactor] logging utils"


---

Part _ of https://github.com/pytorch/pytorch/issues/114548



[ghstack-poisoned]

jon-chuang commented:
@pytorchbot merge

jon-chuang committed:
Update on "[AOT Refactor] logging utils"


---

Part _ of https://github.com/pytorch/pytorch/issues/114548



[ghstack-poisoned]

jon-chuang commented:
@pytorchbot merge

jon-chuang commented:
@pytorchbot merge -f "flakey CI and stacked PRs pass"
